src/AudioPasteDialog.cpp

```diff
@@ -0,0 +1,136 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file AudioPasteDialog.cpp
+
+  @author Vitaly Sverchinsky
+
+**********************************************************************/
+
+#include "AudioPasteDialog.h"
+
+#include <wx/sizer.h>
+#include <wx/checkbox.h>
+#include <wx/stattext.h>
+#include <wx/radiobut.h>
+
+#include "ShuttleGui.h"
+#include "prefs/TracksBehaviorsPrefs.h"
+#include "widgets/WindowAccessible.h"
+
+BEGIN_EVENT_TABLE(AudioPasteDialog, wxDialogWrapper)
+   EVT_BUTTON(wxID_OK, AudioPasteDialog::OnContinue)
+   EVT_BUTTON(wxID_CANCEL, AudioPasteDialog::OnCancel)
+END_EVENT_TABLE()
+
+IMPLEMENT_CLASS(AudioPasteDialog, wxDialogWrapper)
+
+namespace {
+
+const TranslatableString audioPasteModeText[] = {
+   XO("Smart clip\n"
+          "The entire source clip will be pasted into your project, allowing you to access it at anytime."),
+   XO("Selected audio only\n"
+          "Only the selected portion of the source clip will be pasted.")
```

Comment:
These string literals are repeated in TracksBehaviorsPrefs.cpp.  Eliminating duplication might be nice but in this case is not very important.  No bug would result if they were differently modified.  The strings are not used as identifiers.

Reply:
Agreed, though not very clear where exactly such strings could be placed